src/DenseACSets.jl

```diff
@@ -643,6 +702,64 @@ function replace_colons(acs::ACSet, parts::NamedTuple{types}) where {types}
     part == (:) ? (1:nparts(acs, type)) : part
   end)
 end
+# Garbage collection 
+####################
+"""
+Returns the part type of an arbitrary part of an ACSet (we presently assume 
+that all parts are the same type). If there are no objects in the schema, we 
+return nothing
+"""
+function part_type(X::ACSet)
```

Comment:
This is janky; we have the wrappers for a reason. Also, can't this be inferred from the strategy?